### ISSUE TITLE
Custom WP Content URLs ignored for uploads

### ISSUE THREAD

**Issue by adamj88** [bentideswell/magento2-wordpress-integration#74]
If you use a custom content URL set in the `wp-config.php` file, the blog still tries to access the default URLs, e.g: 

```
define('WP_SITEURL', 'https://magento.com/wp');
define('WP_CONTENT_URL', 'https://magento.com/wp-content');
```

The module still tries to access uploads from `https://magento.com/wp/wp-content/uploads/`.

Relevant code: https://github.com/bentideswell/magento2-wordpress-integration/blob/master/Model/App/Url.php#L142-L145

**Comment by bentideswell** [comment]
Hi @adamj88 

I'm working on version 2.0.0.0 of the module and this should be available soon and this will have this fix in.

**Comment by bentideswell** [comment]
This is now available in version 2.0.0.4. You'll need to set WP_CONTENT_URL and WP_CONTENT_DIR.
